packages/hash/frontend/src/components/layout/PageHeader/SearchBar/SearchInput.tsx

```diff
@@ -0,0 +1,144 @@
+import { Box, InputAdornment, InputBase, Tooltip } from "@mui/material";
+import { useRef } from "react";
+import { useKeys } from "rooks";
+import { faXmark } from "@fortawesome/free-solid-svg-icons";
+
+import { SearchIcon, FontAwesomeSvgIcon } from "../../../icons";
+
+export const SearchInput: React.FC<{
+  displayedQuery: string;
+  isMobile: boolean;
+  setResultListVisible: (visible: boolean) => void;
+  setQueryText: (queryText: string) => void;
+}> = ({ displayedQuery, isMobile, setResultListVisible, setQueryText }) => {
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  useKeys(["AltLeft", "KeyK"], () => inputRef.current?.focus());
```

Comment:
If I'm understanding correctly that the keyboard shortcut here is `alt + K`, why was this chosen instead of using `/` as indicated in the designs? Also why `alt` instead of `cmd` on mac?

Reply:
Hey Ben, this is part of the pre-existing Search code. I noticed that `/` in the Search block conflicts with the quick-switch block shortcut `/`, hence I've let it remain as it it. My assumption was this would be taken care of later when we build out the designs for the search results as well (when they're ready). We can discuss what to do with this at standup.

Reply:
In [this internal discussion](https://hashintel.slack.com/archives/C022217GAHF/p1646740022274379) we agreed to use `alt + P` for windows, and `cmd + P` for mac. If you think it makes sense, it'd be good to address this as part of this PR, or in a follow-up task.

Reply:
I think it makes sense to address this as a part of this PR. Will make the change.

Reply:
The shortcut code starts from [here](https://github.com/hashintel/hash/pull/375/files#diff-ebccff6a5e8b22ec9fa2bd595f473d4b51efdf431feaf92d85f3fc3fd6adcccbR102), let me know if it makes sense.